tampermonkey/PPT_AutoComplete.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      3.0
+// @version      4.0
```

Comment:
You will run out of versions at this rate!

Reply:
Are you joking or should we follow a more industry standard approach? :smile:

Reply:
Was joking.  But we are going up in 0.1 increments on our scripts.  TM FTW!